docs/reference/commandline/cli.md

```diff
@@ -48,6 +48,8 @@ by the `docker` command line:
 * `DOCKER_TLS_VERIFY` When set Docker uses TLS and verifies the remote.
 * `DOCKER_CONTENT_TRUST` When set Docker uses notary to sign and verify images.
   Equates to `--disable-content-trust=false` for build, create, pull, push, run.
+* `DOCKER_CONTENT_TRUST_SERVER` The URL of the Notary server to use. This defaults
```

Comment:
I agree with Viktor. We would have to deprecate it anyway, and we can always prefer the env variable over whatever mechanism we come up in the future to configure notary servers.

I'm ok with documenting it, but we can add a note saying that it is likely to be replaced for a a better configuration mechanism in the near future.